.github/workflows/aws-deploy.yml

```diff
@@ -43,16 +38,40 @@ jobs:
           role-session-name: GitHubActions-${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}
           role-duration-seconds: 1200
 
-        # https://github.com/marketplace/actions/aws-cdk-github-actions
-      - name: cdk synth
+      - name: cdk deploy to org-sagebase-dnt-dev (631692904429)
         uses: youyo/aws-cdk-github-actions@v2
         with:
-          cdk_subcommand: 'synth'
-          cdk_args: '--context env=dev --debug'
+          cdk_subcommand: 'deploy'
+          cdk_args: '--require-approval never --context env=dev --progress events --debug'
+          actions_comment: false
+
+  cdk-deploy-prod:
+    runs-on: ubuntu-latest
+    if: ${{ github.ref == 'refs/heads/prod' }}
+    permissions:
+      id-token: write
+      contents: read
+      packages: write
 
-      - name: cdk deploy
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Install dependencies
+        run: pip install -r requirements.txt -r requirements-dev.txt
```

Comment:
small nit, but I think we only need `requirements.txt` here.

Reply:
done. thanks!